keyboards/cest73/tkm/rules.mk

```
@@ -7,7 +7,7 @@ BOOTLOADER = halfkay
 # Build Options
 #   change yes to no to disable
 #
-BOOTMAGIC_ENABLE = full     # Virtual DIP switch configuration
+BOOTMAGIC_ENABLE = lite     # Virtual DIP switch configuration
```

I changed the inline comments in the Bootmagic Full removal PR.

```suggestion
BOOTMAGIC_ENABLE = lite     # Enable Bootmagic Lite
```